kuwala/core/canvas/src/components/Modals/TransformationCatalog/transformation-example-table.css

```diff
@@ -0,0 +1,75 @@
+.transformation-example {
+    border: none;
+}
+
+.transformation-example .ReactTable .rt-tbody {
+    overflow-x: hidden;
+    overflow-y: hidden;
+}
+
+.transformation-example .ReactTable .rt-thead .rt-th{
+    border: none;
+    padding: 0.2rem 0.5rem;
+    background: #8B83BA;
```

Comment:
Should be used from a global color config (also applies to other uses in the file).